Pin Dropper/ViewController.swift

```diff
@@ -10,30 +10,70 @@ import UIKit
 
 class ViewController: UIViewController, GMSMapViewDelegate {
 
+    var pins : [Pin] = []
+    
+    @IBOutlet weak var mapUIView: UIView!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        var camera = GMSCameraPosition.cameraWithLatitude(-33.86,
-            longitude: 151.20, zoom: 6)
-        var mapView = GMSMapView.mapWithFrame(CGRectZero, camera: camera)
+        var camera = GMSCameraPosition.cameraWithLatitude(-33.883633,
+            longitude: 151.193927, zoom: 14)
+        var mapView = GMSMapView.mapWithFrame(CGRectMake(0, 0, self.mapUIView.frame.size.width, self.mapUIView.frame.size.height), camera: camera)
         mapView.myLocationEnabled = true
-        self.view = mapView
+        self.mapUIView.addSubview(mapView)
+        self.view.addSubview(mapUIView)
         mapView.delegate = self
-        
-        var marker = GMSMarker()
-        marker.position = CLLocationCoordinate2DMake(-33.86, 151.20)
-        marker.title = "Sydney"
-        marker.snippet = "Australia"
-        marker.map = mapView
     }
 
     override func didReceiveMemoryWarning() {
         super.didReceiveMemoryWarning()
         // Dispose of any resources that can be recreated.
     }
+    
+   
+    override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject!) {
+        if (segue.identifier == "ToPinsList") {
```

Comment:
Ideally, we want to put the segue identifier `ToPinsList` in a constant string so that we don't mistake it due to a type